src/firefly/js/Firefly.js

```diff
@@ -472,9 +472,11 @@ function handleWebApi(webApiCommands, e, doAppRender) {
             // window.history.pushState('home', 'Home', new URL(window.location).pathname); // ?? is this necessary?
             doAppRender();
             dispatchOnAppReady(() =>  {
-                if (apiCompleted) return;
-                execute?.(cmd,params);
-                apiCompleted= true;
+                setTimeout(() => {
```

Comment:
It seems that `isReady` is fired too early.  I think this is where [the problem](https://github.com/Caltech-IPAC/firefly/blob/1de7779b87999160aa1051750045971deeb21377/src/firefly/js/Firefly.js#L265) is.  That's why doAppRender exited before it's completed.  If possible, fixing the problem there would be better.